testsuite/pom.xml

```diff
@@ -1125,7 +1123,7 @@
                 <arquillianContainer.version>${wildfly25.arquillianContainer.version}</arquillianContainer.version>
 
                 <tests.ignoredCategory>${ignoredCategory.wildfly11}</tests.ignoredCategory>
-                <tests.specialJvmArgs>${specialJvmArgs.wildfly}</tests.specialJvmArgs>
+                <tests.specialJvmArgs></tests.specialJvmArgs>
```

Comment:
well, we could probably delete these all `specialJvmArgs` properties but whatever...

Reply:
let's keep it for now, if we ever need it. It might be useful for JDK17 for example.